catalog/MDCCatalog/MDCCatalogTiles.m

```diff
@@ -3192,7 +3190,7 @@ void MDCCatalogDrawPalettesTile(CGRect frame, NSObject<MDCColorScheme> *colorSch
     label100Style.alignment = NSTextAlignmentLeft;
 
     NSDictionary* label100FontAttributes = @{
-      NSFontAttributeName : [UIFont fontWithName:PalettesFontName size:PalettesFontPointSize],
+      NSFontAttributeName : [UIFont monospacedDigitSystemFontOfSize:PalettesFontPointSize weight:UIFontWeightRegular],
```

Comment:
It might be worth wrapping this font in a static inline function that performs an iOS 9 version check to return the digit font, else returning a system font using a descriptor with a monospace trait.